src/components/Panel/Panel.tsx

```diff
@@ -37,7 +52,18 @@ const Wrapper = styled.div<{
   $hasShadow?: boolean;
   $color?: PanelColor;
   $padding?: PanelPadding;
+  $width?: string;
+  $fillWidth?: boolean;
+  $height?: string;
+  $fillHeight?: boolean;
+  $orientation?: Orientation;
 }>`
+  display: flex;
+  flex-flow: ${({ $orientation = "horizontal" }) =>
+    $orientation === "horizontal" ? "row wrap" : "column"};
```

Comment:
Need to add column wrap too

Reply:
not wrapping columns should be fine as we can keep scrolling. Columns wrap is a bit weird behavior that i think we won't need

Reply:
Please add nowrap for consistency

Reply:
that's the default behavior, i think it's pretty explicit, no?